afpacket/sockopt_linux.go

```diff
@@ -1,3 +1,10 @@
+// Copyright 2012 Google, Inc. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+// +build linux
 package afpacket
```

Comment:
sorry, golint is requesting a space between the build comment and the package line. :)  https://travis-ci.org/google/gopacket/builds/200166979